crates/ursa-metrics/Cargo.toml

```diff
@@ -2,13 +2,15 @@
 name = "ursa-metrics"
 version = "0.1.0"
 edition = "2021"
-authors = ["mtavano <marcello@fleek.co>"]
+authors = ["ossian <oz@fleek.xyz>", "mtavano <marcello@fleek.co>"]
 license = "MIT"
 repository = "https://github.com/Psychedelic/ursa"
 description = "Ursa metrics"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+prometheus = "0.13.2" # used by libp2p_bitswap's internal metrics
```

Comment:
Could we use workspace dependencies here?

Reply:
Sure, although it's only used by the single crate and not a common one.

Reply:
https://github.com/fleek-network/ursa/pull/81/commits/e582e39196e44f3059b640f946f2eb6f638205b3